src/NewHoursReport.java

```diff
@@ -1,131 +1,487 @@
-import java.awt.BorderLayout;
-import java.awt.Desktop;
+import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.*;
-import java.text.NumberFormat.Field;
-import java.util.Iterator;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.List;
+
 import javax.swing.*;
 import javax.swing.filechooser.FileNameExtensionFilter;
 
-import org.apache.poi.*;
-import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.hssf.usermodel.HSSFDateUtil;
 import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.CellType;
 import org.apache.poi.ss.usermodel.CreationHelper;
+import org.apache.poi.ss.usermodel.DataConsolidateFunction;
 import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFCell;
+import org.apache.poi.xssf.usermodel.XSSFRow;
 import org.apache.poi.xssf.usermodel.XSSFSheet;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
 
 
 
-public class NewHoursReport extends JPanel  {
-	JFrame frame;
-	JLabel label;
+
+public class NewHoursReport extends JFrame  {
+	  static JFrame frame;
+	 JLabel label;
```

Comment:
A great convention to get used to early is naming member variables uniquely. For example in my current project a var that is accessible throughout a class would start with `m_` so `label` would become `m_label`. You don't have to use underscores but I would suggest distinguishing these variables in some way.